.github/workflows/scripts/unix_test.sh

```diff
@@ -0,0 +1,13 @@
+TAICHI_REPO_DIR=`pwd`
+TI_LIB_DIR=`python -c "import taichi;print(taichi.__path__[0])" | tail -1`
+TI_LIB_DIR="$TI_LIB_DIR/lib" ./build/taichi_cpp_tests
+export PATH=$TAICHI_REPO_DIR/taichi-llvm/bin/:PATH
+export PATH=$PATH:$HOME/.local/bin
```

Comment:
This line doesn't look right, if you installed without --user, this shouldn't be added to PATH.

Reply:
But this was in every Test section in presubmit.yml.